### PR TITLE
Name Refactoring

### PR DESCRIPTION
**Pull request by RlonRyan**
The goal of this branch is to remove the Names class, as to clean up the code.

### CODE REVIEW

**Comment by RlonRyan** [comment]
I guess the issue with the running PR is that the PR fails if the main branch is failing... but the main branch should probably never be allowed to be failing.

**Comment by RlonRyan** [comment]
The way I have Travis setup, we should keep main the working branch, from which we release, and we work in dev- branches, if that is good with you.

**Comment by RlonRyan** [comment]
I think that should help some...

File: src/main/java/com/InfinityRaider/AgriCraft/utility/RegisterHelper.java [hunk 1]
```diff
@@ -34,15 +38,34 @@ public static void registerBlock(Block block, String name, Class<? extends ItemB
...
-                if (fruit != null && fruit.getItem() != null && !OreDictHelper.hasOreId(fruit, Names.Objects.crop + name)) {
-                    OreDictionary.registerOre(Names.Objects.crop + name, fruit);
+                if (fruit != null && fruit.getItem() != null && !OreDictHelper.hasOreId(fruit, "crop" + name)) {
+                    OreDictionary.registerOre("crop" + name, fruit);
                 }
             }
         }
     }
+	
+	public static void registerItemRenderer(Item item) {
```
Comment (InfinityRaider) [inline src/main/java/com/InfinityRaider/AgriCraft/utility/RegisterHelper.java]: This will need a SideOnly(Side.CLIENT) annotation

Comment (RlonRyan) [inline src/main/java/com/InfinityRaider/AgriCraft/utility/RegisterHelper.java]: Ok

**Comment by RlonRyan** [comment]
And I think that this way you could theoretically commit to the branches I work on, for edits such as that...

**Comment by InfinityRaider** [comment]
Ah yes, I didn't notice it was from the dev branch my apologies.

**Comment by RlonRyan** [comment]
Should I try a merge?

**Comment by InfinityRaider** [comment]
Go for it.

**Comment by RlonRyan** [comment]
It seems to have worked well!

**Comment by InfinityRaider** [comment]
In the future, if we want to do something new, just create a new branch and make a PR for it. This is the most useful and clear imo.

**Comment by RlonRyan** [comment]
So you like this setup?

**Pull request merged**